cms/forms/wizards.py

```diff
@@ -21,16 +23,13 @@
     text_widget = forms.Textarea
 
 
-class SlugWidget(forms.widgets.TextInput):
-    """
-    Special widget for the slug field that requires Title field to be there.
-    Adds the js for the slugifying.
-    """
-    class Media:
-        js = (
-            'admin/js/urlify.js',
-            static_with_version('cms/js/dist/bundle.forms.slugwidget.min.js'),
-        )
+class SlugWidget(AdminSlugWidget):
+    """Compatibility shim with deprecation warning:
+    SlugWidget has moved to cms.admin.forms"""
+    def __init__(self, *args, **kwargs):
+        warnings.warn("Import SlugWidget from cms.admin.forms. SlugWidget will be removed from cms.forms.wizzards",
```

Comment:
Typo on `cms.forms.wizzards`, should read `cms.forms.wizards`

Reply:
Changed! Thanks! (I tend to write wizard with double z 🤦‍♂️)